tests/math.test.ts

```
@@ -0,0 +1,99 @@
+
+  it('should handle negative numbers', () => {
+    expect(roundTo(-10.12345, 2)).toBe(-10.12);
+    expect(roundTo(-10.125, 2)).toBe(-10.13);
+  });
+
+  it('should handle zero value', () => {
+    expect(roundTo(0, 5)).toBe(0);
+  });
+
+  it('should handle large numbers', () => {
+    expect(roundTo(123456789.123456789, 5)).toBe(123456789.12346);
```

Biome (2.1.2)

[error] This number literal will lose precision at runtime.

The value at runtime will be 123456789.12345679

(lint/correctness/noPrecisionLoss)